src/Exporter.php

```diff
@@ -96,7 +96,7 @@ public function shortenedRecursiveExport(array &$data, ?RecursionContext $proces
     public function shortenedExport(mixed $value): string
     {
         if (is_string($value)) {
-            $string = str_replace("\n", '', $this->export($value));
+            $string = str_replace("\n", '', $this->exportString($value));
```

Comment:
this means we take a shortcut from

`export -> recursiveExport -> exportString` to just a single `exportString`